src/main/services/dataCollectors/modifiedLinesCollector/ModifiedMethodsHelper.groovy

```diff
@@ -24,6 +24,9 @@ class ModifiedMethodsHelper {
         List<String> diffJOutput = runDiffJ(ancestorFile, targetFile);
         List<String> textualDiffOutput = runTextualDiff(ancestorFile, targetFile);
 
+        targetFile.delete()
```

Comment:
@rafaelmotaalves could not see how the commit message actually matches what was done here

Reply:
My bad, I did this to make because I noticed that this files were not properly removed after while I was doing creating the tests. Should I rebase it?